tests/test_scylla.c

```diff
@@ -136,20 +137,54 @@ void test_scylla(void) {
     set_transaction_timestamp(transaction, (int64_t)timestamps[i]);
     set_transaction_value(transaction, (int64_t)i);
     /* insert test input transactions to ScyllyDB */
-    insert_transaction_into_bundleTable(session, transaction, 1);
-    insert_transaction_into_edgeTable(session, transaction, 1);
+    insert_transaction_into_bundleTable(&db_client_service, transaction, 1);
+    insert_transaction_into_edgeTable(&db_client_service, transaction, 1);
   }
 
   /* test select bundles by edge in edge table */
-  test_get_column_from_edgeTable(session);
+  test_get_column_from_edgeTable(&db_client_service);
   /* test find transactions by bundles or addresses */
-  test_get_transactions(session);
+  test_get_transactions(&db_client_service);
 
   free_scylla_iota_transaction(&transaction);
-  cass_cluster_free(cluster);
-  cass_session_free(session);
+  db_client_service_free(&db_client_service);
+  scylla_api_logger_release();
 }
 
+void test_db_identity_table(void) {
+  db_client_service_t db_client_service;
+
+  const char hashes[][NUM_FLEX_TRITS_HASH] = {
+      {TRANSACTION_1}, {TRANSACTION_1}, {TRANSACTION_2}, {TRANSACTION_2}, {TRANSACTION_3}, {TRANSACTION_3},
+  };
+  const char expected_select_hashes[][NUM_FLEX_TRITS_HASH] = {
+      {TRANSACTION_2}, {TRANSACTION_2}, {TRANSACTION_1}, {TRANSACTION_1}, {TRANSACTION_3}, {TRANSACTION_3},
+  };
+
+  TEST_ASSERT_EQUAL_INT(ta_logger_init(), SC_OK);
+  scylla_api_logger_init();
```

Comment:
The `scylla_api_logger_init` can be wrapped into our own functions. Let's do it in another pull request.